public/index.html

```diff
@@ -24,6 +24,34 @@
       .card-header.btn:focus {
         box-shadow: none;
       }
+
```

Comment:
Not urgent for this PR, but at some point we should pull these `style` attributes out into actual CSS, probably.